app/main.py

```diff
@@ -1 +1,18 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    command, file, directory = command.split()
+    if command == "mv":
+        if "/" in command:
+            command_ls = directory.split("/")
+            for direction in command_ls[:-1]:
+                file_directory = directory[: directory.index(direction)]
+                os.mkdir(f"{file_directory}{direction}")
+
+            with open(file, "r") as first_file, open(directory, "w") as copy:
+                copy.write(first_file.read())
+
+            os.remove(file)
+        else:
+            os.rename(file, directory)
```

Comment:
will 13-16 lines do the same thing as rename, if there is won't be "/" in command ? No need to use else, here, try to reuse 13-26 lines